2-mandatory/2-currency-code-lookup.js

```diff
@@ -19,8 +19,15 @@ const COUNTRY_CURRENCY_CODES = [
 
 function createLookup(countryCurrencyCodes) {
   // write code here
+  return Object.assign(...COUNTRY_CURRENCY_CODES.map((d) => ({ [d[0]]: d[1] }))
```

Comment:
instead of using d (as a variable name) , always use a variable name with meaningful name